frontend/packages/rhoas-plugin/src/topology/components/KafkaNode.scss

```diff
@@ -0,0 +1,3 @@
+.co-m-resource-kafka-connection {
+  background-color: --pf-global--palette--red-300;
+}
```

Comment:
This is somewhat some automatic feature/magic that puts colors on AKC labels - if it remove that things  

So pattern for those is `.co-m-resource-{{name-of-your-crd}}` but I'm not sure if there is some flag to avoid this

Reply:
Please add a comment to this please.
I know we plan to improve the way we assign colors with a future extetnsion.